fenix/app/src/main/java/org/mozilla/fenix/translations/automatic/AutomaticTranslationItem.kt

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations.automatic
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.mozilla.fenix.R
+import java.util.Locale
+
+/**
+ * AutomaticTranslationItem that will appear on Automatic Translation screen.
+ *
+ * @property languageLocale The text that will appear in the list.
+ * @property automaticTranslationOption The option that the user selected.
+ */
+@Parcelize
+data class AutomaticTranslationItem(
+    val languageLocale: Locale,
```

Comment:
Please adjust to a simple string for now, will ultimately probably become `Language`.